e2e/test/scenarios/collections/collections.cy.spec.js

```diff
@@ -178,12 +178,26 @@ describe("scenarios > collection defaults", () => {
     });
   });
 
-  describe("render last edited by when names are null", () => {
-    beforeEach(() => {
-      restore();
-      cy.signInAsAdmin();
-    });
+  it('should support markdown in collection description', () => {
+    cy.request("PUT", "/api/collection/9", {
+      description: "# header"
+    })
+
+    visitRootCollection();
+
+    cy.get('table').within(() => {
+      cy.findByText("First collection").parent().parent().within(() => {
+        cy.icon("info").trigger("mouseenter"); 
+      })
```

Comment:
Do we really need to grab `parent().parent()` here? Are there more than one "info" icons in scope of `findByText("First collection")`

Reply:
yup, almost every line contains an icon, I need first, but if we change the structure of tests, it will fail if we don't use specific line. I'll take a look if it can be simplified

Reply:
used `closest('tr')` instead